climada/entity/impact_funcs/base.py

```diff
@@ -273,3 +273,70 @@ def set_sigmoid_impf(self, *args, **kwargs):
         LOGGER.warning("The use of ImpactFunc.set_sigmoid_impf is deprecated."
                        "Use ImpactFunc.from_sigmoid_impf instead.")
         self.__dict__ = ImpactFunc.from_sigmoid_impf(*args, **kwargs).__dict__
+
+    @classmethod
+    def from_poly_s_shape(
+        cls,
+        intensity: tuple[float, float, float],
+        threshold: float,
+        half_point: float,
+        upper_limit: float,
+        exponent: float,
+        haz_type: str,
+        impf_id: int = 1,
+        **kwargs):
+        """S-shape polynomial impact function hinging on four parameter.
+
+        f(I) = D(I)**exponent / (1 + D(I)**exponent) * upper_limit
+
+        D(I) = max[I - threshold, 0] / (half_point - threshold)
+
+        This function is inspired by Emanuel et al. (2011)
+        https://doi.org/10.1175/WCAS-D-11-00007.1
+
+        Parameters
+        ----------
+        intensity: tuple(float, float, float)
+            tuple of 3 intensity numbers along np.arange(min, max, step)
+        threshold : float
+            Intensity threshold below which there is no impact.
+            Should in general be larger than 0 for computational efficiency
+            of impacts.
+        half_point : float
+            Intensity at which 50% of maxixmum impact is expected.
+            Must be larger than thres.
+        upper_limit : float
+            Maximum impact value. Must be larger than 0.
+        exponent: float
+            Exponent of the polynomial. Must be larger than 0.
```

Comment:
Indeed, must is again too strong.

Reply:
I made the exponent to be positive. The function is senseless otherwise. Negative exponents now raises an error.

Reply:
But an exponent of exactly zero would work, right? The function would just be constant then (thinking about calibration: It might be better to have as few restrictions as possible to avoid setting strange parameter bounds)